### ISSUE TITLE
Update failure 4.3.1

### ISSUE THREAD

**Issue by ingvar-nikiforov** [esm-bundle/chai#23]
I am using this repo in my project
```
"chai": "npm:@esm-bundle/chai@^4.3.0"
```

Running `npm run update` fails with this error message:

```
git:(master) npm update

> @esm-bundle/chai@4.3.1 postinstall src_path/node_modules/chai
> husky install

sh: husky: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! @esm-bundle/chai@4.3.1 postinstall: `husky install`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the @esm-bundle/chai@4.3.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by web-padawan** [comment]
Looks like the major bump of `husky` was added to the patch release (see #18).
Please consider reverting that commit and publishing `4.3.2` and then `5.0.0` with a proper fix.

Also note, husky 5 is currently experimental and contains a lot of breaking changes.

**Comment by joeldenning** [comment]
husky 5 isn't experimental - the problem here is that the husky-4-to-5 script that we ran on this repo left one step for us to do manually that we forgot to do. I have fixed it in f68d6deacd8c913a746427b0d7d44b554203dc5a and released the fix in https://github.com/esm-bundle/chai/releases/tag/v4.3.1-fix.0. Let me know if you run into any issues using that version.

**Comment by web-padawan** [comment]
Thanks for taking care of this issue.

> husky-4-to-5 script that we ran on this repo left one step for us to do manually that we forgot to do

Based on this and discussions like https://github.com/typicode/husky/issues/868 I must admit that husky 5 has some problems to solve.
